### PR TITLE
Worldwind 2.2.1 Update

### PR DESCRIPTION
**Pull request by FJBDev**
WorldWind 2.2.1 [Release notes](https://github.com/NASAWorldWind/WorldWindJava/releases/tag/v2.2.1) as of 06/07/2022

@wolfgang-ch Maybe it's better to wait for the 22.**NEXT** ? Less risks ? I will let you decide.

### CODE REVIEW

**Comment by wolfgang-ch** [comment]
> @wolfgang-ch Maybe it's better to wait for the 22.**NEXT** ? Less risks ? I will let you decide.

No risk, no fun -> no fun

**Comment by wolfgang-ch** [comment]
This PR contains these errors, the code will fail with a new MT instance

image

**Comment by FJBDev** [comment]
> This PR contains these errors, the code will fail with a new MT instance
> 

Fixed

File: bundles/net.tourbook/src/net/tourbook/map3/layer/tourtrack/TrackPathOptimized.java [hunk 1]
```diff
@@ -572,12 +570,12 @@ protected void doDrawOutlineVBO(final DrawContext dc, final int[] vboIds, final
...
          if (!isPickingMode && config.isShowDirectionArrows) {
-            drawDirectionArrows(dc, vboIds, pathData);
+            drawDirectionArrows(dc, vboIds);
          }
 
       } finally {
-         gl.glBindBuffer(GL2.GL_ARRAY_BUFFER, 0);
-         gl.glBindBuffer(GL2.GL_ELEMENT_ARRAY_BUFFER, 0);
+         gl.glBindBuffer(GL.GL_ARRAY_BUFFER, 0);
+         gl.glBindBuffer(GL.GL_ELEMENT_ARRAY_BUFFER, 0);
```
Comment (wolfgang-ch) [inline bundles/net.tourbook/src/net/tourbook/map3/layer/tourtrack/TrackPathOptimized.java:578]: Why are many GL* constants changed into GL ?

This is a very complex code (at least for me) and should not be modified when not needed

Comment (wolfgang-ch) [reply bundles/net.tourbook/src/net/tourbook/map3/layer/tourtrack/TrackPathOptimized.java]: > Maybe an Eclipse optimization ?

I don't now how you changed it, there are also conversions from `4` to `4L`, this seems also to be automated

**Pull request merged**